docs/design/libraries/DllImportGenerator/StructMarshalling.md

```diff
@@ -250,7 +265,7 @@ struct HRESULT
 
 In this case, the underlying native type would actually be an `int`, but the user could use the strongly-typed `HResult` type as the public surface area.
 
-> :question: Should we support transparent structures on manually annotated blittable types? If we do, we should do so in an opt-in manner to make it possible to have a `Value` property on the blittable type.
+> :question: Should we support transparent structures on manually annotated types that wouldn't need marshalling otherwise types? If we do, we should do so in an opt-in manner to make it possible to have a `Value` property on the type without assuming that it is for interop in all cases.
```

Comment:
```suggestion
> :question: Should we support transparent structures on manually annotated types that wouldn't need marshalling otherwise? If we do, we should do so in an opt-in manner to make it possible to have a `Value` property on the type without assuming that it is for interop in all cases.
```